1.6/Dockerfile.rhel7

```diff
@@ -46,9 +45,11 @@ RUN yum install -y tar zip unzip java-1.7.0-openjdk jenkins-1.609.1 && \
     mv oc /usr/local/bin && \
     rm oadm && \
     rm openshift && \
-    chmod -R og+rw /opt/openshift
+    mkdir -p /opt/openshift && chmod -R og+rw /opt/openshift && \
+    mkdir -p /usr/share/jenkins && chmod -R og+rw /usr/share/jenkins
 
-VOLUME ["/var/lib/jenkins"]
+COPY ./contrib/openshift /opt/openshift
+COPY ./contrib/jenkins /usr/local/bin
```

Comment:
same comment I made to @rhcarvalho.  moving these COPYs after the chmod of the dir means the files in the dir aren't getting chmod'd.  that seems potentially problematic.